Revert/Sources/HomeViewController.swift

```diff
@@ -57,9 +57,10 @@ final class HomeViewController: UITableViewController {
   }
 
   // MARK: Private
+  typealias HomeItems = [[HomeItem]]
```

Comment:
I am a bit confused about `[[HomeItem]]` and `[HomeSectionItem]`, they are both generated by `RevertItems.home.newData()`, do we need both?

Reply:
Thanks so much for pointing out! `HomeSectionItem` was created to avoid nested array. But looks like I forgot to replace it on this class.